tests_and_backend_clean_build.sh

```diff
@@ -7,8 +7,6 @@ set -e
 
 mkdir build && cd build
 
-cmake -DCMAKE_BUILD_TYPE=Debug -DBUILD_TESTS=ON ../modules/backend/
+cmake -DCMAKE_BUILD_TYPE=Debug -DBUILD_TESTS=1 ..
```

Comment:
Is it enough to build and run tests at once?
From what I understand it only switches to build tests or not. 

If I am correct I think we need to keep line 14 'ctest --output-on-failure', because this script is meant to run tests with build at once.